### PR TITLE
chore: example-app for MAF integration test

### PR DESCRIPTION
**Pull request by insignias**
Issue #, if available:

Description of changes:
example-app for MAF integration test

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [x] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: workbench-core/example/app/src/generateRouter.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      credentials: true
+    })
+  );
+  // parse application/json
+  app.use(express.json());
+  app.use(cookieParser());
+  app.use(csurf('none'));
+
+  app.use(verifyToken(authenticationService, { ignoredRoutes: staticRoutesIgnored, loggingService: logger }));
+  app.use(withAuth(authorizationService, { logger: logger }));
```
[the remark this answers is not in the document]
Comment (insignias) [reply workbench-core/example/app/src/generateRouter.ts:54]: Added a fix in the app/src/generateRouter.ts

File: workbench-core/example/app/src/routes/authRoutes.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { wrapAsync } from '../utilities/errorHandlers';
+
+export function setUpAuthRoutes(router: Router, auth: AuthenticationService, logger: LoggingService): void {
+  // Get auth provider's login URL with temporary state and PKCE strings
+  router.get('/login', wrapAsync(getAuthorizationCodeUrl(auth)));
+
+  // User would have manually logged in at this point, and received an auth code. Exchange auth code for token
+  router.post(
+    '/token',
+    wrapAsync(getTokensFromAuthorizationCode(auth, { loggingService: logger, sameSite: 'none' }))
```
[the remark this answers is not in the document]
Comment (insignias) [reply workbench-core/example/app/src/routes/authRoutes.ts:26]: Added a fix in the app/src/generateRouter.ts

File: workbench-core/example/infrastructure/src/constructs/encryotionKeyWithRotation.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
```
Comment (bhudnell) [inline workbench-core/example/infrastructure/src/constructs/encryotionKeyWithRotation.ts:3]: file name has a typo

File: workbench-core/example/infrastructure/src/constructs/encryotionKeyWithRotation.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+ */
+
+/* eslint-disable no-new */
+import { Aws, CfnOutput } from 'aws-cdk-lib';
+import { AccountPrincipal, PolicyDocument, PolicyStatement } from 'aws-cdk-lib/aws-iam';
+import { Key } from 'aws-cdk-lib/aws-kms';
+import { Construct } from 'constructs';
+
+export interface EncriptionKeyWithRotationProps {
```
Comment (bhudnell) [inline workbench-core/example/infrastructure/src/constructs/encryotionKeyWithRotation.ts:12]: nit: typo

**Review by bhudnell — APPROVED** [review]

**Review by SanketD92 — APPROVED** [review]

**Pull request merged**